tests/mocha.js

```diff
@@ -1,4 +1,9 @@
 /* eslint-disable no-console */
+
+require('@babel/register')({
+  extensions: ['.ts', '.js'],
+});
```

Comment:
When running tests we need a way to compile source-code on the fly, since there isn't a build step in-between. Thats what this babel hook is for.